microsoft/testsuites/performance/storageperf.py

```diff
@@ -60,7 +60,7 @@ class StoragePerformance(TestSuite):
         priority=3,
         timeout=TIME_OUT,
         requirement=simple_requirement(
-            min_core_count=72,
+            min_core_count=2,
```

Comment:
The core requirements avoid side effects from other VMs on the same host. What's the reason to change it to 2?

Reply:
To test new SKUs which can have vCPUs < 72. Some times even 4.

Reply:
I agree with your comment about dedicated hosts and colocation effect. But there are scenarios where we need to test new VM SKUs which are as small as 4. In cases like these irrespective of how many VMs are on the same host, we should be able to test the VM IO perf. Its not fair to exclude these VMs because they might collocated with other VMs. Its up to the user to deal with/investigate performance issues in case of colocation effects. IMHO LISAv3 should not decide this.

Reply:
I have seen fio failing on lower core counts. @SRIKKANTH Can you please test on a 2 core and 4 core machine and check if this works?

Reply:
@squirrelsc I don't think its true in case of LISAv2. 
@LiliDeng Can you confirm? 
@somil55 I could run FIO on 2&4 core VMs without any issues. I can provide you a recent job if you want.

Reply:
V2 doesn't block it in code, but in practice, we only run and collect data on designed perf VM sizes, and view the data in report. If the perf is dropped, it's definitely need to be fixed. If we run test on all VMs, it's hard to know where the perf issue is from, so it's not actionable.
In V3, it's limited on test code, and in future, we should add an automated way to measure the perf, instead of viewing the data. If we allow test on shared VMs, it will produce noise, and not automatable.

Reply:
@squirrelsc In that case how we are supposed to user v3 for IO perf validation in VM SKU validation where we need to test all the VM sizes <>=72 vCPUs.

Reply:
The current logic can test vm size, which core count equals or is greater than 72. But it's not a good solution, a feature called "ExclusiveVM" should be implemented to pickup the VMs. It's not implemented yet. So the 72 core is temp solution.

Reply:
I'm still not clear why we cant use v3 for non exclusive VMs or how can we test non exclusive VMs. Should we maintain our own branch for new SKU validations?

Reply:
Again, the perf result on a shared VM is not actionable. We cannot use the test results on a shared VM to file a performance bug.

Reply:
Thats what we are doing so far in case of new VM SKU validation and everyone is ok with it.